packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/scheduler.controller.js

```diff
@@ -70,8 +70,12 @@ export default /* @ngInject */ function TelephonySchedulerCtrl(
 
     return self.scheduler
       .getEvents({
-        'dateStart.from': start.format(),
-        'dateEnd.to': end.format(),
+        'dateStart.to': moment(start)
```

Comment:
I think you switched dateStart.to and dateEnd.from values

Reply:
dateEnd.from should be 1st day of period, while dateStart.to should be last day of period

Reply:
and it's not the start and the end of the month
I think the original code is OK, just switch values between dateStart.from and dateEnd.to